### ISSUE TITLE
Pathfinding breaks when multiple tokens are being dragged

### ISSUE THREAD

**Issue by Jakimcio** [caewok/fvtt-elevation-ruler#137]
The pathfinding can be broken accidentally when you try to move a different token while another is being moved. This does not happen instantly or always but at some point it seems to just happen, as long as pathfinding is involved (I tried for a while to cause it on a blank map w/o any walls to pathfind around). This causes me as the gm to only be able to pull the tokens within their field of vision, as can be seen here as I try to drag a token through a wall (it will stop at the wall):

![image](https://github.com/caewok/fvtt-elevation-ruler/assets/130793238/1163f75e-dbe4-41b9-8337-65a6c417c151)

Using Find the Culprit I deactivated all mods but Elevation Ruler and libWrapper and it persists.

```
Foundry Virtual Tabletop: Version 12 Stable, 12.328
Game System: pf2e, 6.0.4
Active Modules: 3
Performance Mode: 3

OS: Windows 10.0.0 x86 (64-bit)
Client: Not(A:Brand/24.0.0.0
GPU: ANGLE (AMD, AMD Radeon RX 5700 (0x0000731F) Direct3D11 vs_5_0 ps_5_0, D3D11)
Max Texture Size: 16384

Scene: 12000 x 9000 | Grid Size: 200 | Padding Percentage: 0.25
Walls: 590 | Ambient Lights: 5 | Ambient Sounds: 0 | Tiles: 0 | Tokens: 17
Largest Texture: 8000 x 5800.78125 

Actors: 114 | Items: 1 | Journal Entries: 14 | Rollable Tables: 0 |
Playlists: 2 | Compendium Packs: 84 | Chat Messages: 423

World Scripts: 
```

The error in console when this happens (which increases whenever you try and drag a token) is: 

```
cdt2d_bundle.js:587 Uncaught TypeError: Cannot read properties of undefined (reading 'upperIds')
    at mergeHulls (cdt2d_bundle.js:587:26)
    at monotoneTriangulate (cdt2d_bundle.js:641:7)
    at cdt2d (cdt2d_bundle.js:128:15)
    at cdt2dConstrainedGraph (cdt2d_access_functions.js:50:21)
    at Pathfinder._buildTriangles (pathfinding.js:197:23)
    at Pathfinder.initialize (pathfinding.js:181:10)
    at Pathfinder.runPath (pathfinding.js:288:52)
    at calculatePathPointsForSegment (segments.js:99:19)
    at Ruler._getMeasurementSegments (segments.js:59:7)
    at 🎁CONFIG.Canvas.rulerClass.prototype._getMeasurementSegments#0 (libWrapper-wrapper.js:187:52)
    at Ruler.measure (foundry.js:35951:26)
    at Ruler._onMouseMove (foundry.js:36564:10)
    at TokenPF2e._onDragLeftMove (Token.js:67:33)
    at 🎁CONFIG.Token.objectClass.prototype._onDragLeftMove#0 (libWrapper-wrapper.js:187:52)
    at MouseInteractionManager.callback (foundry.js:30691:17)
    at #handleDragMove (foundry.js:31070:26)
    at #handlePointerMove (foundry.js:31035:64)
    at vh.notifyListeners (EventBoundary.ts:1464:33)
    at vh.notifyTarget (EventBoundary.ts:658:14)
    at vh.propagate (EventBoundary.ts:310:18)
    at vh.dispatchEvent (EventBoundary.ts:210:14)
    at vh.mapPointerMove (EventBoundary.ts:815:64)
    at vh.mapEvent (EventBoundary.ts:231:28)
    at ra.onPointerMove (EventSystem.ts:396:31)
```

**Comment by Jakimcio** [comment]
It can be fixed by viewing (not even activating) another scene and then returning.

**Comment by revilowaldow** [comment]
This is the same error as https://github.com/caewok/fvtt-elevation-ruler/issues/101

**Comment by caewok** [comment]
Closed b/c duplicate of #101.
